### ISSUE TITLE
Issues with hard pinning dependencies

### ISSUE THREAD

**Issue by UebelAndre** [alexliesenfeld/httpmock#23]
Can you actually not hard pin everything? I didn't realize what affect this would have downstream when you first mentioned it in #21 

```output
error: failed to select a version for `regex`.
    ... required by package `httpmock v0.5.1`
    ... which is depended on by `cargo-raze v0.6.1 (/Users/user/Code/cargo-raze/impl)`
versions that meet the requirements `=1.3.9` are: 1.3.9

all possible versions conflict with previously selected packages.

  previously selected package `regex v1.4.1`
    ... which is depended on by `cargo-clone-crate v0.1.4`
    ... which is depended on by `cargo-raze v0.6.1 (/Users/user/Code/cargo-raze/impl)`

```

**Comment by alexliesenfeld** [comment]
Yes, indeed. Sorry for the trouble (it's late in Germany). I'll remove exact version pinning and will update all dependencies.

**Comment by UebelAndre** [comment]
Kein Problem! Vielen Dank für die harte Arbeit!! 😃

**Comment by UebelAndre** [comment]
You could probably unpin everything for now. The library was usable again after #20 was merged. If that would be easier. Updating dependencies can be a pain sometimes 😅

**Comment by alexliesenfeld** [comment]
Yes, it's already worked in that way. Just waiting for CI test results again.

**Comment by alexliesenfeld** [comment]
This should be fixed with #22
